tests/test_models.py

```diff
@@ -63,16 +63,16 @@ def test_instantiate_inventory_record(self):
         self.assertEqual(record.restock_level, 2)
 
     def test_inventory_serialize(self):
-        record = Inventory(product_id=1, name="monitor", condition=Inventory.Condition.NEW,
-                            quantity=10, reorder_quantity=20, restock_level=2)
+        record = Inventory(product_id=1, name="monitor", condition=Inventory.Condition.NEW, quantity=10, reorder_quantity=20, restock_level=2, active=False)
```

Comment:
Can we check for pylint here?

Reply:
I have run make lint on the entire file. There are no errors